web/src/user_group_popover.ts

```diff
@@ -99,13 +107,14 @@ export function toggle_user_group_info_popover(element, message_id) {
     );
 }
 
-export function register_click_handlers() {
+export function register_click_handlers(): void {
     $("#main_div").on("click", ".user-group-mention", (e) => {
         e.stopPropagation();
 
         const $elt = $(e.currentTarget);
         const $row = $elt.closest(".message_row");
-        const message = message_lists.current.get(rows.id($row));
+        assert(message_lists.current);
+        const message = message_lists.current.get(rows.id($row)!)!;
```

Comment:
I assumed here that `rows.id` and `message_lists.current.get` will always be defined and non-null, since this handler is called when a user clicks on a mention pill from a message.

Reply:
This is not a performance-sensitive hot loop, and the reasoning required to establish that these values aren’t undefined or null is complicated and very nonlocal, so for either or both of those reasons, don’t use a non-null assertion (`!`).